application/config/locales/connectors/dataverse/en.yml

```diff
@@ -116,6 +119,8 @@ en:
         page_title: "OnDemand Loop - Dataverse Dataset"
     landing_page:
       installation_items:
+        button_submit_text: "Search"
+        field_search_placeholder: "Search installations"
```

Comment:
I believe that the landing page has the same functionality than collections. So if collections is filter, this should be filter. If search, then search.